routes/api/contacts.js

```diff
@@ -1,25 +1,105 @@
-const express = require('express')
+const express = require("express");
+const router = express.Router();
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  removeContact,
+  updateContact,
+} = require("../../models/contacts");
 
-const router = express.Router()
+router.get("/", async (__, res) => {
+  try {
+    const contacts = await listContacts();
+    res.json({
+      status: "success",
+      code: 200,
+      data: contacts,
+    });
+  } catch (error) {
+    res.status(400).json({
+      status: "error",
+      code: 400,
+      message: "Bad Request",
+    });
+  }
+});
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/:contactId", async (req, res) => {
+  try {
+    const { contactId } = req.params;
+    const contact = await getContactById(contactId);
+    res.json({
+      status: "success",
+      code: 200,
+      data: contact,
+      message: "Contact has been found",
+    });
+  } catch (error) {
+    res.status(400).json({
+      status: "error",
+      code: 400,
+      message: "Bad Request",
+    });
+  }
+});
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.post("/", async (req, res) => {
+  try {
+    const body = req.body;
+    const contact = await addContact(body);
```

Comment:
```suggestion
    const contact = await addContact(req.body);
```
Też dobrze, a mniej pisania. :)